plugin/xcode.vim

```diff
@@ -94,7 +99,13 @@ function! s:open(path)
 endfunction
 
 function! s:switch(target)
-  execute '!sudo xcode-select -s' . s:cli_args(a:target)
+  if has('nvim')
```

Comment:
Is it worth extracting this if/else to `s:runner` and using it here and in `s:default_runner_command`?